torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2089,6 +2112,28 @@ def _make_tensor_helper(shape, low=None, high=None):
                         # see discussion https://github.com/pytorch/pytorch/pull/47761#issuecomment-747316775
                         SkipInfo('TestGradients', 'test_fn_gradgrad', device_type='cuda'),)
                     ),
+    OpInfo('linalg.det',
+           op=torch.linalg.det,
```

Comment:
Add 'det' as an alias like this:

https://github.com/pytorch/pytorch/blob/36ce673f16269724f629f2a1897b8720ba6c8f18/torch/testing/_internal/common_methods_invocations.py#L1820

Reply:
done, 6ff2599c87